doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,228 @@
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1288">#1288</a></td>
+<td valign="top"><code>SharedClassCacheInfo</code> no longer contains a <code>JVMLEVEL</code> constant</td>
+<td valign="top">OpenJDK10 and later: All platforms</td>
+<td valign="top"><code>SharedClassCacheInfo.getCacheJVMLevel()</code> used to return the <code>JVMLEVEL</code> constant that maps to a Java version number, for example <code>JVMLEVEL_JAVA8</code>. This call now returns only the Java version number, for example <code>10</code> for Java 10.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1370">#1370</a>
+<!-- (<a href="http://openjdk.java.net/jeps/318">JEP318</a>) --></td>
+<td valign="top">New <code>nogc</code> GC policy</td>
+<td valign="top">All platforms</td>
+<td valign="top">Under this policy, no memory is reclaimed through garbage collection. The Java object heap is expanded until an <code>OutOfMemory</code> error is generated and the VM shuts down.<br/>Enable this policy with the <b>-Xgcpolicy:nogc</b> or <b>-XX:+UseNoGC</b> options.<br/>You might use this mode for testing or for short-lived applications. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support is now available</td>
+<td valign="top">Linux, Windows</td>
+<td valign="top">Ability to diagnose problems with the VM, garbage collector, or JIT.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1699">#1699</a></td>
+<td valign="top">Specify heap limits as a percentage of total memory</td>
+<td valign="top">All platforms</td>
+<td valign="top">Ability to easily set heap limits with the <b>-XX:MaxRAMPercentage</b> and <b>-XX:InitialRAMPercentage</b> options instead of statically setting them with the <b>-Xmx</b> and <b>-Xms</b> options.</td>
+</tr>
+<!-- Remove? Confirm.<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1776">#1776</a></td>
+<td valign="top">Extra page of memory no longer required when specifying an object heap size that is a multiple of the page size</td>
+<td valign="top">Linux on Z and z/OS</td>
+<td valign="top">This change avoids wasting memory or calculating a heap size that is 16 bytes less than the page size.</td>
+</tr>-->
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1668">#1668</a></td>
+<td valign="top">Idle tuning feature enabled</td>
+<td valign="top">Linux on Z, Linux on Power systems</td>
```

Comment:
Please clarify these are additions to the xLinux support that was already available.